### PR TITLE
refactor(miniapp): DOMA-9911 updated the miniapp according to the new template

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
Just remove old `miniapp` and generate it with `createapp` again 

image

### CODE REVIEW

**Comment by Alllex202** [comment]
7 thousand lines just to show one label, ok 😂

Do you really need to update the migration?

**Comment by nomerdvadcatpyat** [comment]
> Do you really need to update the migration?

Probably no. I will delete this change

File: .sonarcloud.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
 sonar.cpd.exclusions=**/*.js,**/*.ts,**/OldBaseNewsForm.tsx
-sonar.exclusions=**/*.test.js,**/*.spec.js
```
Comment (Alllex202) [inline .sonarcloud.properties:2]: Are you sure you should do this?

Comment (nomerdvadcatpyat) [reply .sonarcloud.properties:2]: Yes, i need to disable duplicates check in miniapp app because it's very similar as miniapp template

Comment (Alllex202) [reply .sonarcloud.properties:2]: But this will disable all checks.. Оr not?

Is it possible to skip just duplicates checks?

Comment (Alllex202) [reply .sonarcloud.properties:2]: Try using `sonar.cpd.exclusions `
https://docs.sonarsource.com/sonarcloud/advanced-setup/analysis-scope/#scope-of-duplication-detection

**Review by Alllex202 — APPROVED** [review]

**Review by toplenboren — APPROVED** [review]

**Review by abshnko — APPROVED** [review]

**Review by pahaz — APPROVED** [review]

File: apps/miniapp/middlewares/oidc.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                await req.session.save()
+
+                return res.redirect(nextUrl)
+            } catch (err) {
+                await req.session.destroy()
+                return next(err)
+            }
+        })
+
+        app.set('trust proxy', true)
```
Comment (pahaz) [inline apps/miniapp/middlewares/oidc.js:183]: it should be inside prepareKeystoneApp

**Pull request merged**